test/integration/bun-types/bun-types.test.ts

```diff
@@ -422,71 +420,20 @@ describe("@types/bun integration test", () => {
       });
 
       expect(emptyInterfaces).toEqual(expectedEmptyInterfacesWhenNoDOM); // should still have no empty interfaces
-      expect(diagnostics).toEqual([
```

Comment:
The line number/character is useful to detect regressions/changes in where TypeScript reports errors (for example this did actually happen very recently in 5.9 which affected serve-types.test.ts)

Reply:
It was a change in TypeScript itself, not necessarily a bug anywhere. I've found it useful to know about and having it in ci tells us automatically. Having as much information from the compiler is useful for us to know when things break, without it, there could be subtle changes we don't detect in CI but would require us to adjust bun-types